README.md

```diff
@@ -265,6 +265,72 @@ This is a general procedure on how to do it:
 
 Both of these monitoring base classes are dashboard segments, so you can add them to your monitoring by calling `.addSegment()` on the `MonitoringFacade`.
 
+### Custom dashboards
+
+If you want *even* more flexibility, you can take complete control over dashboard generation by leveraging dynamic dashboarding features. This allows you to create an arbitrary number of dashboards while configuring each of them separately. You can do this in three simple steps:
+
+1. Create the dynamic dashboard factory
+2. Create `IDynamicDashboardSegment` implementations
+3. Add Dynamic Segments to MonitoringFacade
+
+#### **Create the dynamic dashboard factory**
+
+The below code sample will generate two dashboards with the following names:
+* ExampleDashboards-HostedService
+* ExampleDashboards-Infrastructure
+
+
+```
+  // create the dynamic dashboard factory.
+  const factory = new DynamicDashboardFactory(stack, "DynamicDashboards", {
+    dashboardNamePrefix: "ExampleDashboards",
+    // a list of dashboard names is the minimum required config information
+    // for creating dynamic dashboards
+    dashboardConfigs: [
+      { name: "HostedService" },
+      { name: "Infrastructure" },
```

Comment:
Make one of these show a few more options as an example.

Reply:
Fixed.